docs/content/en/docs/reference/baremetal/netbooting-and-tinkerbell.md

```diff
@@ -0,0 +1,350 @@
+
+As someone deploying an EKS Anywhere cluster on Bare Metal, you have several opportunities to interact with Tinkerbell:
+
+* **Create a hardware CSV file**: You are required to create a [hardware CSV file]({{< relref "./bare-preparation/#prepare-hardware-inventory" >}}) that contains an entry for every physical machine you want to add at cluster creation time. 
+* **Create an EKS Anywhere cluster**: By modifying the [Bare Metal configuration file ]({{< relref "../clusterspec/baremetal" >}}) used to create a cluster, you can change some Tinkerbell settings or add actions to define how the operating system on each machine is configured.
+* **Monitor provisioning**: You can follow along with the [Tinkerbell Overview]({{< relref "#overview-of-tinkerbell-in-eks-anywhere" >}}) in this page to monitor the progress of your hardware provisioning, as Tinkerbell finds machines and attempts to PXE boot, configure, and restart them.
+
+## Using Tinkerbell on EKS Anywhere
+
+The sections below step through how Tinkerbell is integrated with EKS Anywhere to deploy a Bare Metal cluster.
+Refer to the [Tinkerbell Documentation](https://docs.tinkerbell.org/) for further information on Tinkerbell.
+
+### Create bare metal CSV file
+
+The information that Tinkerbell uses to provision machines for the target EKS Anywhere cluster needs to be gathered in a CSV file with the following format:
+
+```
+hostname,bmc_ip,bmc_username,bmc_password,mac,ip_address,netmask,gateway,nameservers,labels,disk
+eksa-cp01,10.10.44.1,root,PrZ8W93i,CC:48:3A:00:00:01,10.10.50.2,255.255.254.0,10.10.50.1,8.8.8.8,type=cp,/dev/sda
+...
+```
+
+Each physical, bare metal machine is represented by a comma-separated list of information on a single line.
+It includes information needed to identify each machine (the NIC’s MAC address), PXE boot the machine, point to the disk to install on, and then configure and start the installed system.
+See [Preparing hardware inventory]({{< relref "./bare-preparation/#prepare-hardware-inventory" >}}) for details on the content and format of that file. 
+
+### Modify the cluster specification file
+
+Before you create a cluster using the [Bare Metal configuration]({{< relref "../clusterspec/baremetal" >}}) file, you can make Tinkerbell-related changes to that file.
+In particular, [TinkerbellDatacenterConfig fields]({{< relref "../clusterspec/baremetal/#tinkerbelldatacenterconfig-fields" >}}), [TinkerbellMachineConfig fields]({{< relref "../clusterspec/baremetal/#tinkerbellmachineconfig-fields" >}}), and [Tinkerbell Actions](https://docs.tinkerbell.org/actions/action-architecture/) can be added or modified.
+
+Tinkerbell actions vary based on the operating system you choose for your EKS Anywhere cluster.
+Actions are stored internally and not shown in the generated cluster specification file, so you must add those sections yourself to change from the defaults.
```

Comment:
Put a link here that points to where the default template actions are listed for Ubuntu and BR?